bin/martinize2

```diff
@@ -972,8 +973,14 @@ def entry():
         itp_paths = []
         # Merge chains if required.
         if args.merge_chains:
-            for chain_set in args.merge_chains:
-                vermouth.MergeChains(chain_set).run_system(system)
+            if "all" not in args.merge_chains:
+                input_chain_sets = [i.split(",") for i in args.merge_chains]
+                for chain_set in input_chain_sets:
+                    vermouth.MergeChains(chains=chain_set, all_chains=False).run_system(system)
+            elif "all" in args.merge_chains and len(args.merge_chains) == 1:
+                vermouth.MergeChains(chains=None, all_chains=True).run_system(system)
+            else:
+                LOGGER.warning("Multiple conflicting merging arguments given. Please check input arguments.")
```

Comment:
I think this can be stronger, and raise an ArgumentError explaining the conflict